plugins/modules/packaging/os/zypper.py

```diff
@@ -128,6 +128,13 @@
         description:
           - Adds C(--replacefiles) option to I(zypper) install/update command.
         version_added: '0.2.0'
+    clean_deps:
+        type: bool
+        required: false
+        default: false
+        description:
+          - Adds C(--clean-deps) option to I(zypper) remove command.
+        version_added: '4.5.0'
```

Comment:
4.5.0 has been released today, so the next version this can be added is 4.6.0:
```suggestion
        version_added: '4.6.0'
```

Reply:
Ok, sorry about that. Thanks for the review!

Reply:
No problem!